examples/cactus-check-connection-ethereum-validator/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "check-connection-ethereum-validator",
+  "version": "1.0.0-rc.3",
+  "private": true,
+  "scripts": {
+    "start": "node ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "debug": "nodemon --inspect ../../dist/packages/cactus-cmd-socketio-server/src/main/typescript/routing-interface/www.js",
+    "build": "npm run build-ts && npm run copy-static-assets && npm run copy-blp-config && npm run replace-blp-config-path",
+    "build-ts": "tsc -p ./tsconfig.json",
+    "tslint": "tslint -c tslint.json -p tsconfig.json './*.ts'",
+    "copy-static-assets": "ts-node copyStaticAssets.ts",
+    "copy-blp-config": "ts-node copyBLPConfig.ts",
+    "replace-blp-config-path": "ts-node replaceBLPConfigPath.ts",
+    "init-check-connection-ethereum-validator": "ln -s ../examples/cactus-check-connection-ethereum-validator/node_modules ../../dist/node_modules"
+  },
+  "dependencies": {
+    "@types/node": "^14.0.24",
+    "socket.io": "^2.0.4",
```

Comment:
@StepniowskiP Could socketio 4 be used instead by any chance? (in general where there's no reason to use older versions I prefer the newest just in case vulnerabilities are found in the older versions down the line which might not get backported fixes)

Reply:
When I try to update socketio to 4.0.0 or even 3.0.0 I get parser error in Verifier. So I guess we have to stay with this version for now.

Reply:
@StepniowskiP Is that parser error due to the verifier using an older version as well - and if yes would it be solved by upgrading it there as well and then trying again?

Reply:
Yes, it actually works. After updating Verifier's socket.io dependency version to 4.4.0 everything works as intended.

Reply:
@StepniowskiP That is great news! In that case if you don't mind, please update the versions either here or open a separate issue+PR with the change. I'm fine with any one of those options.

Reply:
@StepniowskiP Great, thank you very much!